src/db/tables/emailAddresses.js

```diff
@@ -69,6 +69,7 @@ async function getEmailAddressRecordByEmail (email) {
 // Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
 /* c8 ignore start */
 async function addSubscriberUnverifiedEmailHash (user, email) {
+  const lowerCaseEmail = email.toLowerCase()
```

Comment:
This will help in the future, but we still have email addresses in the DB that are not lower case that won't work right now correct?

Should we migrate these? Might be a big migration... could we file a separate ticket to investigate this? We'd have to recalculat  their sha1 hashes (I am pretty sure HIBP lower-cases so they aren't matching anything anyway).

Reply:
@rhelmer thanks! Yes. I'm thinking we should just use a one-off script to migrate all the existing records. I will get a ticket going